packages/web-client/app/templates/index.hbs

```diff
@@ -1,36 +1,50 @@
-<main class="cardstack-container" data-test-cardstack>
-  <header class="cardstack-section-header">
-    <h1 class="cardstack-section-header__title" data-test-cardstack-header>
+<Boxel::Dashboard
+  {{did-insert (fn this.updateEdges)}}
+  @darkTheme={{true}}
+  data-test-cardstack-dashboard
+>
+  <:body>
+  <div class="cardstack-container">
+    <div class="cardstack-section-header">
+    <h2 class="cardstack-section-header__title" data-test-cardstack-dashboard-header>
       Cardstack
-    </h1>
-    {{!-- <Boxel::Button
+    </h2>
+    {{!-- {{#unless this.cardstackSession.isAuthenticated}} --}}
+    <Boxel::Button
       @kind="secondary-dark"
       @size="tall"
-      data-test-cardstack-connect-button
+      {{!-- {{on "click" this.cardstackSession.login}} --}}
+      data-test-cardstack-login-button
     >
       Connect Account
-    </Boxel::Button> --}}
-
-    <div class="cardstack-landing-hero">
-      <img src={{@model.logo}} alt="" role="presentation" class="cardstack-landing-hero__logo">
-      <p class="cardstack-landing-hero__description">The world’s first cooperative SaaS platform.</p>
-      <p class="cardstack-landing-hero__description">Own the platform. Choose the tools. Work your way.</p>
-      <LinkTo @route="index" class="cardstack-landing-hero__cta">
-        Learn more {{svg-jar "caret-thin-right" class="cardstack-landing-hero__cta-icon"}}
-      </LinkTo>
+    </Boxel::Button>
+    {{!-- {{/unless}} --}}
```

Comment:
This will be same as the Layer 2 connect button. Please remove references to any cardstack session. I'm not sure which design we'll use, so remove this button for now.

Reply:
Hmm, on this particular page isn't this button correct?